go.mod

```diff
@@ -22,13 +22,13 @@ require (
 	k8s.io/gengo v0.0.0-20190813173942-955ffa8fcfc9 // indirect
 	k8s.io/klog v0.4.0
 	sigs.k8s.io/cluster-api v0.0.0-20190822050630-23ef7a8a48c2
-	sigs.k8s.io/controller-runtime v0.2.0-rc.0
+	sigs.k8s.io/controller-runtime v0.2.0
 	sigs.k8s.io/controller-tools v0.2.0-rc.0
```

Comment:
This can go to v0.2.0 as well

Reply:
tools?

Reply:
Yup! https://github.com/kubernetes-sigs/controller-tools/tree/v0.2.0